tests/api2/test_pool_dataset_quota_alert.py

```diff
@@ -1,17 +1,13 @@
-#!/usr/bin/env python3
-
 # License: BSD
 
-import sys
-import os
 import re
 import pytest
 from pytest_dependency import depends
```

Comment:
Let's put a single space in between builtin imports, 3rd party imports and finally local imports